modules/xfem/src/userobjects/GeometricCut2DUserObject.C

```diff
@@ -209,3 +197,19 @@ GeometricCut2DUserObject::crossProduct2D(const Point & point_a, const Point & po
 {
   return (point_a(0) * point_b(1) - point_b(0) * point_a(1));
 }
+
+Real
+GeometricCut2DUserObject::cutFraction(unsigned int cut_num, Real time) const
+{
+  Real fraction = 0.0;
+
+  if (time >= _cut_time_ranges[cut_num].first)
```

Comment:
Really should have a mooseAssert here to validate both the input and the RHS access.